tests/EppoClientTest.php

```diff
@@ -244,48 +267,6 @@ public function testHandlesLoggingException()
         $this->assertEquals('control', $assignment);
     }
 
-    /**
-     * @param array $subjects
-     * @param string $experiment
-     * @return array
-     * @throws HttpRequestException
-     * @throws InvalidApiKeyException
-     * @throws InvalidArgumentException
-     * @throws GuzzleException
-     * @throws \Psr\SimpleCache\InvalidArgumentException
-     */
-    private function getAssignments(array $subjects, string $experiment): array
```

Comment:
This and the next function were just folded into the `testGetAssignmentVariationAssignmentSplits()` test to clean things up